packages/react-dom/src/client/ReactDOMFiberTextarea.js

```diff
@@ -86,14 +89,13 @@ export function initWrapperState(element: Element, props: Object) {
     }
   }
 
-  var value = props.value;
-  var initialValue = value;
+  let initialValue = props.value;
```

Comment:
Hmm, wonder why it was written the other way initially. 😛

Reply:
te-hehe

Do you recommend putting it back?
I think it does not make sense to keep this line. I guess it was written for better readability. Since it is the same reference.

What do you think @clemmy

Reply:
I think this is probably fine.